Tests/Builder/FormContractorTest.php

```diff
@@ -46,4 +47,68 @@ public function testGetFormBuilder()
             $this->formContractor->getFormBuilder('test', array('foo' => 'bar'))
         );
     }
+
+    public function testDefaultOptionsForSonataFormTypes()
+    {
+        $admin = $this->getMockBuilder('Sonata\AdminBundle\Admin\AdminInterface')->getMock();
```

Comment:
Not sure why (can't see a conflict in [the Composer manifest](https://packagist.org/packages/phpunit/phpunit#4.8.26) ) but that does feel oddly familiar…